src/modules/modals/EditRow.vue

```diff
@@ -17,6 +17,10 @@
 					type="error">
 					{{ t('tables', '"{columnTitle}" should not be empty', { columnTitle: column.title }) }}
 				</NcNoteCard>
+				<NcNoteCard v-if="localRow[column.id] && column.type === 'text-link' && !isValidUrlProtocol(localRow[column.id])"
```

Comment:
Something separate but we could think about how we can reduce the code duplication between CreateRow and EditRow